client/state/jetpack-connect/constants.js

```diff
@@ -1,4 +1,4 @@
-export const JETPACK_CONNECT_TTL = 60 * 60 * 1000; // 1 hour
-export const JETPACK_CONNECT_TTL_SECONDS = JETPACK_CONNECT_TTL / 60;
+export const JETPACK_CONNECT_TTL_SECONDS = 60 * 60; // 1 hours in seconds
```

Comment:
Just a minor typo in the inline comment here.

```suggestion
export const JETPACK_CONNECT_TTL_SECONDS = 60 * 60; // 1 hour in seconds
```